### DISCUSSION TITLE
Plan with Minecraft in Docker?

### DISCUSSION THREAD

**Discussion by obvionaoe** [plan-player-analytics/Plan#2400] [Archive]
### Question

Hi!

How difficult would it be to use Plan with a Minecraft server running inside a container?

I currently run a Minecraft Fabric server using the [itzg/minecraft-server](https://github.com/itzg/docker-minecraft-server) image and it would be awesome if there was a way to use it with Plan and keep Plan's webserver running inside that container!

Thanks

**Comment by rymiel** [comment]
Plan doesnt have a separate web server, it is part of the plugin and run by the minecraft server, so theoretically all it needs is a bound port

**Comment by AuroraLS3** [comment]
Yes Plan can be run in docker - as a bunch of people are running it with Pterodactyl which runs the minecraft server inside a container.

You'll need to open a port to the docker container though, as rymiel said. How you do this depends on how you're running the container - if you run it using `docker run` adding `-p 8804:8804` binds the default Plan port to the host machine.
